### PR TITLE
Conversion to the gang now places your gang info in your notes.

### PR DESCRIPTION
**Pull request by harmonyn**
Fun fact: If you forget which gang you get converted to, there is otherwise ZERO way to find out without having to track down a fellow gangster or hope that you have spray paint to test it yourself.

### CODE REVIEW

**Comment by Lumipharon** [comment]
This always really fucked me off.
:+1:

File: code/game/gamemodes/gang/gang.dm [hunk 1]
```diff
@@ -224,6 +225,7 @@ var/list/gang_colors_pool = list("red","orange","yellow","green","blue","purple"
 				gangster_mind.current.Paralyse(5)
 				gangster_mind.current.visible_message("<FONT size=3><B>[gangster_mind.current] looks like they've given up the life of crime!<B></font>")
 			gangster_mind.current << "<FONT size=3 color=red><B>You have been reformed! You are no longer a gangster!</B><BR>You try as hard as you can, but you can't seem to recall any of the identities of your former gangsters...</FONT>"
+			gangster_mind.memory = ""
```
Comment (KazeEspada) [inline code/game/gamemodes/gang/gang.dm]: Won't this wipe traitor objectives?

Comment (bgobandit) [inline code/game/gamemodes/gang/gang.dm]: Probably, but I don't know that there's another way. (This is the same thing that cult does upon deconversion.)

Comment (KazeEspada) [inline code/game/gamemodes/gang/gang.dm]: You could always just add a new memory stating you got deganged.

**Pull request merged**